sample.env

```diff
@@ -1,12 +1,13 @@
 #RERUM API access keys
-ACCESS_TOKEN=token
-REFRESH_TOKEN=short-token
+ACCESS_TOKEN=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Ik9FVTBORFk0T1RVNVJrRXlOREl5TTBFMU1FVXdNMFUyT0RGQk9UaEZSa1JDTXpnek1FSTRNdyJ9.eyJodHRwOi8vc3RvcmUucmVydW0uaW8vYWdlbnQiOiJodHRwczovL3N0b3JlLnJlcnVtLmlvL3YxL2lkLzY1MzgxNmQzNjY1NzI0YWJmZDEwMmRlNSIsImh0dHA6Ly9yZXJ1bS5pby91c2VyX3JvbGVzIjp7InJvbGVzIjpbImR1bmJhcl91c2VyX3B1YmxpYyIsImdsb3NzaW5nX3VzZXJfcHVibGljIiwibHJkYV91c2VyX3B1YmxpYyIsInJlcnVtX3VzZXJfcHVibGljIl19LCJodHRwOi8vZHVuYmFyLnJlcnVtLmlvL3VzZXJfcm9sZXMiOnsicm9sZXMiOlsiZHVuYmFyX3VzZXJfcHVibGljIiwiZ2xvc3NpbmdfdXNlcl9wdWJsaWMiLCJscmRhX3VzZXJfcHVibGljIiwicmVydW1fdXNlcl9wdWJsaWMiXX0sImlzcyI6Imh0dHBzOi8vY3ViYXAuYXV0aDAuY29tLyIsInN1YiI6ImF1dGgwfDY1MzgxNmQxZTdhYTBlMzM1NjQ2NWY1MCIsImF1ZCI6Imh0dHA6Ly9yZXJ1bS5pby9hcGkiLCJpYXQiOjE2OTg2ODcwNzYsImV4cCI6MTczMDI0NDY3NiwiYXpwIjoiNjJKc2E5TXhIdXFoUmJPMjBnVEhzOUtwS3I3VWU3c2wiLCJzY29wZSI6Im9mZmxpbmVfYWNjZXNzIn0.3b5qHqwseGCjR7Xgr8s7XZO_5jybWjIBM9aWEpl6njS7YCdMhwTydE7fOotOjAm21J1DfEuRyBZD-zSJ8MxTlnuzRLE8gUvK8G-6jWgv5k8ezu7w1V1JlfKH25wIkuzIgLDkpnmGloLnH-FO-TaJ_gIis4DscQUKjACVs616nIiO7fD0eVLZpEbamwdFatuGdZTYYDxxqZzIaeVyYYYbwd8ye0HIJGWO1z2IEVPS0sRZSqc9DYTm_M4DBok90wWH6tZfFAZhsJdjjlizA6Gwhwv5rvv3nMJjbqyfgwBWWIrerrGcWu-9ny9Qy-N7MBdxbOin-3vgP2ri17QBJPyPSA
```

Comment:
oo never commit this stuff to the repo.  Someone can perform a man in the middle attack with this information.  This is supposed to be in a local `.env` file and nowhere else.